sisl/viz/plotly/input_fields/dropdown.py

```diff
@@ -0,0 +1,231 @@
+import numpy as np
+
+from sisl import Spin
+from .._input_field import InputField
+
+
+class DropdownInput(InputField):
+
+    _type = 'dropdown'
+
+    _default = {
+        "width": "s100% m50% l33%",
+        "params": {
+            "placeholder": "Choose an option...",
+            "options": [
+            ],
+            "isMulti": False,
+            "isClearable": True,
+            "isSearchable": True,
+        }
+    }
+
+    def __init__(self, *args, **kwargs):
+
+        # Build the help string
+        params = kwargs.get("params")
```

Comment:
`kwargs.get("params")` may fail if not provided in `kwargs`, did you intent `kwargs.get("params", None)`?

Reply:
I recently realised that `kwargs.get("params")` == `kwargs.get("params", None)` != `kwargs["params"]` (and same for `kwargs.pop`). Is this a new feature of dicts that doesn't work with previous versions of python? Or am I missing something and they are not exactly equivalent?

Reply:
My bad, you are correct `.get(<>) == .get(<>, None)`, I had always thought that `get` and `__getitem__` were the same for no arguments. 

Probably clarify that you want `None` returned. But maybe I should just get used to use the default. :)

It is fine as is, then!